tests/test_types_generator.py

```diff
@@ -170,8 +170,12 @@ def bucket_data_tagging(expiry: str = "unset", sensitivity: str = "unset") -> Bu
     return BucketDataTagging(expiry=expiry, sensitivity=sensitivity)
 
 
-def bucket_encryption(enabled: bool = False, type: Optional[str] = None) -> BucketEncryption:
-    return BucketEncryption(enabled=enabled, type=type)
+def bucket_encryption(enabled: bool = False, key: str = "", type: Optional[str] = None) -> BucketEncryption:
+    return BucketEncryption(enabled=enabled, key=key, type=type)
+
+
+def bucket_kms_key(id: str = "", rotation_enabled: bool = False) -> Key:
```

Comment:
There already was a utility in `test_types_generator` to create `Key` instances, it feels like this one is redundant when the existing one can be used instead.